### PR TITLE
Fix "webroot not found" when installed to the root of the webserver

### PR DESCRIPTION
**Pull request by icewind1991**
See https://github.com/nextcloud/server/pull/6452#issuecomment-329386726 and https://github.com/nextcloud/server/pull/6495#issuecomment-329393087

`''` is a valid webroot

### CODE REVIEW

File: lib/private/Template/ResourceLocator.php [hunk 1]
```diff
@@ -171,7 +171,7 @@ protected function append($root, $file, $webRoot = null, $throw = true) {
 		if (!$webRoot) {
```
Comment (MorrisJobke) [inline lib/private/Template/ResourceLocator.php]: Why not here as well?

Comment (icewind1991) [reply lib/private/Template/ResourceLocator.php]: done

**Review by skjnldsv — APPROVED** [review]
Perfect!

**Review by rullzer — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Regression: this breaks the server if installed in a subfolder: #6607

**Pull request merged**